apps/ensindexer/src/config/types.ts

```diff
@@ -69,18 +69,18 @@ export interface ENSIndexerConfig {
    * Invariant:
    * - The URL must be a valid URL. localhost urls are allowed (and expected).
    */
-  ensRainbowEndpointUrl: string;
+  ensRainbowEndpointUrl: URL;
```

Comment:
`EndpointUrl` is redundant, let's use `ensRainbowUrl` to match the existing pattern (`ensAdminUrl`, `ensNodePublicUrl`, etc)